docs/detections/visual-event-analyzer.asciidoc

```diff
@@ -109,9 +114,20 @@ Events are categorized based on the `event.category` value.
 [role="screenshot"]
 image::images/event-type.png[]
 
-When you select an `event.category` pill, all the events within that category are listed in the left panel. To view more details about a specific event, select it from the list.
+When you select an `event.category` pill, all the events within that category are listed in the left panel. To display more details about a specific event, select it from the list.
 
 [role="screenshot"]
 image::images/event-details.png[]
 
 NOTE: In {stack} versions 7.10.0 and newer, there is no limit to the number of events that can be associated with a process. However, in {stack} versions 7.9.0 and earlier, each process is limited to only 100 events.
+
+To examine alerts on the process, select the pill that shows the total number of alerts associated with it (*_x_ alerts*). You can then click on an alert in the left pane to display its details.
```

Comment:
```suggestion
To examine alerts on the process, select the pill that shows the total number of alerts associated with it (*_x_ alert*). You can then click on an alert in the left pane to display its details.
```

nit: I changed this from plural to singular to be consistent with the other event types